modules/apps/data-engine/data-engine-test/src/testFunctional/macros/DERenderer.macro

```diff
@@ -65,6 +65,21 @@ definition {
 		}
 	}
 
+	macro assertDataInLinktoPageField {
+		if (!(isSet(index))) {
+			var index = "1";
+		}
+
+		var selectedPage = selenium.getElementValue("xpath=(//label[normalize-space(text())='${webContentLinkToPageFieldLabel}']/following-sibling::div//div[contains(@class,'input-group')]//input[contains(@type,'text')])[${index}]");
```

Comment:
@guivilarim, why are you using a path here instead of add it to a dedicated file `*.path`?